test/tests/coord_type/tests

```diff
@@ -32,4 +32,12 @@
     cli_args = 'Problem/rz_coord_axis=X Outputs/file_base=coord_type_rz_integrated_x_rotation_out'
     requirement = "The system shall support cylindrical coordinates with the radial direction in the 'y' direction with discontinous Galkerin and integrated boundary conditions."
   [../]
+  [rz_general]
+    type = Exodiff
+    input = coord_type_rz_general.i
+    exodiff = coord_type_rz_general.e
+    requirement = "The system shall support cylindrical coordinates with a general axis."
+    design = 'Mesh/index.md'
+    issues = '#24339'
+  []
```

Comment:
can we do a csv diff?

Reply:
Is it your philosophy to just do CSVDiff when Exodiff is not strictly necessary? In this particular case, I think an Exodiff is advantageous since we can check the whole distribution.

Reply:
In perusing it seemed like the postprocessors seemed like what you really cared about, but if you care about a field, then go with Exodiff!

Reply:
Yeah that "T" little guy is in that show list, just hard to see.